.github/workflows/reusable-macos.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     name: build and test (${{ inputs.os }})
     timeout-minutes: 60
     env:
+      FORCE_COLOR: 1
```

Comment:
Non-blocking: we could put this in a top-level block, for consistency with the other `reusable-*` workflows?

Reply:
Sure, although `reusable-ubuntu.yml` already has it at the job level, but I'll move that as well.